src/components/Home/Home.module.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../scss/bootstrap.scss";
+
+.avoidOverlapWithStickyHeader::before {
+  content: " ";
+  display: block;
+  position: relative;
+  height: 58px; // minimal height of navbar
+  margin-top: -58px;
```

Comment:
The height of the header changes responsively with the font size (due to RFS). Set the height of `::before` element to the minimum header height to avoid gaps.